main.go

```diff
@@ -161,7 +164,7 @@ func main() {
 			}
 		}
 	})
-	r.Run("0.0.0.0:8081")
+	r.Run("0.0.0.0:8083")
 }
 
 func parsePullRequest(conf *config, action string, pr *github.PullRequestEvent) []buildOptions {
```

Comment:
It seems to me that the logic in this function to decide on publishArtifacts build parameter is broken, as `buildContainers` will never be true. What is the historical reason for this? Should I fix or clean it up?

Reply:
Just clean it up. IIRC it was an attempt at auto-pushing containers, but it never worked as intended.

Reply:
Thanks, I cleaned it up. I leave it as an independent commit so that it can be reviewed if we ever need this feature properly implemented.